qiita_db/support_files/patches/26.sql

```diff
@@ -0,0 +1,52 @@
+-- June 11,, 2015
+-- Adds portal type to analyses
```

Comment:
Please, improve description of the patch. It's doing a lot more things that just adding portal type to the analysis object.